index.js

```diff
@@ -123,8 +125,11 @@ Plugin.prototype.make = function(compilation, callback) {
 
 		var dep = new SingleEntryDependency(entry);
 		compilation.addEntry("", dep, path.relative(this.basePath, file).replace(/\\/g, "/"), function() {
+		        if(dep.module && dep.module.error && dep.module.error.error && dep.module.error.error.stack) {
+                                this.log.warn(dep.module.error.error.stack);
```

Comment:
`this` probably references the wrong `this` here as it's inside an anon function call